packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -20,7 +20,8 @@ const SUPPORTED_PROVIDERS = fs
 // returns the content of index.js with import statements added
 const addWebImports = (content, imports) => {
   const importStatements = imports.map((imp) => {
-    return `import ${imp.import} from '${imp.from}'`
+    const importWithFrom = imp.import === null ? '' : `${imp.import} from`
+    return `import ${importWithFrom} '${imp.from}'`
   })
```

Comment:
If my suggestion about changing the imports data-structure works out, then you could remove these lines.